python/templates/JuliaCollection.jl.jinja2

```diff
@@ -1,2 +1,13 @@
 include("{{ class.bare_type }}.jl")
-{{ class.bare_type }}Collection = Vector{ {{ class.bare_type }}{% if params_jl %}{ {{ params_jl | join(',') }} }{% endif %} }
+using .{{ class.bare_type }}Module: {{class.bare_type}}
+
+{% macro julia_parameters(params) %}
+{%- if params_jl -%}{
+{%- set comma = joiner(',') -%}
+{%- for par in params -%}
+{{ comma() }}Main.{{ par }}Struct
+{%- endfor -%}
+}{%- endif -%}
+{% endmacro %}
+
+{{ class.bare_type }}Collection = Vector{ Main.{{ class.bare_type }}Struct{{ julia_parameters(params_jl) }} }
```

Comment:
What is the purpose of this, since there is also a `Collection.jl.jinja2` template now?

Reply:
This was a mistake.